mview.c

```diff
@@ -237,7 +247,7 @@ static void update_tables(struct MailboxView *mv)
   m->changed = false;
   m->msg_flagged = 0;
   padding = mx_msg_padding_size(m);
-  for (i = 0, j = 0; i < m->msg_count; i++)
+  for (i = 0, j = 0; i < m->email_max; i++)
```

Comment:
Why this change?

Reply:
I've explained that in the commit message, here
https://github.com/neomutt/neomutt/pull/3762/commits/7eef19ede1a19751840b468cc94181a5ede109c6

Reply:
Thanks - yep this is also what #3765 is about. I'm not sure we ever want msg_count to exceed the number of emails, so I'm not yet sure what the solution for *that* is going to be.